### PR TITLE
Contributor Demographic Diversity

### PR DESCRIPTION
**Pull request by Benyuel**
Based on the languages removed,added or byte per language changed in the pr diff, the contributor gets a multiplier towards their own vote in order to encourage diversity within the democracy.

### CODE REVIEW

File: github_api/prs.py [hunk 1]
```diff
@@ -312,3 +312,12 @@ def post_status(api, urn, sha, state, description):
...
     }
     api("POST", path, json=data)
+
+
+def get_languages(api, urn, pr_num):
+    """ yield the difference in language diversity between this pr & master """
+    """ List languages for the specified repository. 
+        The value on the right of a language is the 
+        number of bytes of code written in that language. """
+    path = get_pr(api, urn, pr_num)['head']['repo']['url'] + '/languages'
```
Comment (hongaar) [inline github_api/prs.py:322]: why not re-use the `pr` object passed into `get_votes()`?

**Comment by hongaar** [comment]
Like the implementation, but not sure if this is something which should count towards your vote weight, especially not with a 1:3 ratio...

**Comment by mikesmiffy128** [comment]
This is definitely not something which should count. All this does is incentivise people to clutter the repository with random code in as many languages as possible for no reason. :-1:

**Comment by Benyuel** [comment]
@mikesmiffy128: that's called chaos

**Comment by mikesmiffy128** [comment]
Okay, but language diversity doesn't achieve anything. We're not even talking about culture here - that's a whole other debate - but this is literally just challenging people to shoehorn in code that fewer people will understand, probably making democracy more difficult due to the added effort of understanding multiple obscure languages.

There is literally no benefit to doing this.

Edit: sorry I have no sense of fun

**Pull request closed without merging**